pom.xml

```diff
@@ -226,6 +226,8 @@
                                 <ts.global.s2i.quarkus.native.builder.image>${ts.global.s2i.quarkus.native.builder.image}</ts.global.s2i.quarkus.native.builder.image>
                                 <!-- Services used in test suite -->
                                 <postgresql.latest.image>${postgresql.latest.image}</postgresql.latest.image>
+                                <!-- FIXME: if the same image name is used in two test classes, the second one fails -->
```

Comment:
@jsmrcka FYI. This is a fix for failing tests. Unfortunately, I was not able to reproduce this issue in the Framework

Reply:
@fedinskiy This is so weird. Anyway, it feels like we should have an issue (in FW?) for this. Or at least a trello card. Or is it already known / reported?